packages/third-party/tailwind-preset/src/lynx.ts

```diff
@@ -2,13 +2,14 @@
 // Licensed under the Apache License Version 2.0 that can be found in the
 // LICENSE file in the root directory of this source tree.
 
-import { display, position, underline } from './plugins/lynx/index.js';
+import { display, position, textDecoration } from './plugins/lynx/index.js';
 
 /**
  * Should be used with Tailwind JIT and configured with purge, otherwise the bundle will be too large.
  */
 export default {
-  plugins: [position, underline, display],
+  content: ['./**/*.{js,ts,jsx,tsx,css}'],
```

Comment:
```suggestion
  content: ['./**/*.{js,ts,jsx,tsx}'],
```

Reply:
Resolved over chat: the preset shouldn't include `content` at all --- AI did this :)